plotting/ft_plot_cloud.m

```diff
@@ -94,14 +100,11 @@ function ft_plot_cloud(pos, val, varargin)
 radius             = ft_getopt(varargin, 'radius', 4 * ft_scalingfactor('mm', unit));
 rmin               = ft_getopt(varargin, 'rmin',   1 * ft_scalingfactor('mm', unit));
 scalerad           = ft_getopt(varargin, 'scalerad', 'yes');
+cmap               = ft_getopt(varargin, 'colormap', 'parula');
```

Comment:
why do you think I added the following section 

-if ft_platform_supports('parula')
 -  cmap             = ft_getopt(varargin, 'colormap', 'parula');
 -else
 -  cmap             = ft_getopt(varargin, 'colormap', 'jet');
 -end

removing it ensures that the code (once more) breaks on MATLAB versions prior to 2014.